.changeset/tough-dots-visit.md

```diff
@@ -0,0 +1,6 @@
+---
+"@markings/output-html": minor
+"@markings/react": minor
```

Comment:
```suggestion
"@markings/react": major
```

Reply:
That would cause these packages to go to v1. Are you sure?